scalafix-tests/input/src/main/scala/test/escapeHatch/AnnotationScopes.scala

```diff
@@ -0,0 +1,273 @@
+/*
+rules = [
+  "class:scalafix.test.EscapeHatchDummyLinter"
+]
+*/
+package test.escapeHatch
+
+object AnnotationScopes {
+
+  // No suppressing
+  type DummyType_0 = Any // assert: EscapeHatchDummyLinter
+
+  trait DummyTrait_0 { // assert: EscapeHatchDummyLinter
+    def aDummy = () // assert: EscapeHatchDummyLinter
+  }
+
+  object DummyObject_0 { // assert: EscapeHatchDummyLinter
+    def aDummy = () // assert: EscapeHatchDummyLinter
+  }
+
+  class DummyClass_0( // assert: EscapeHatchDummyLinter
+                 val aDummy: Int, // assert: EscapeHatchDummyLinter
+                 val bDummy: Int) { // assert: EscapeHatchDummyLinter
+
+    val cDummy = 0 // assert: EscapeHatchDummyLinter
+    var dDummy = 0 // assert: EscapeHatchDummyLinter
+
+    def this(eDummy: String, // assert: EscapeHatchDummyLinter
+             fDummy: String) { // assert: EscapeHatchDummyLinter
+      this(eDummy.toInt, fDummy.toInt) // assert: EscapeHatchDummyLinter
+    }
+
+    def gDummy( // assert: EscapeHatchDummyLinter
+              hDummy: Int, // assert: EscapeHatchDummyLinter
+              iDummy: Int): Unit = { // assert: EscapeHatchDummyLinter
+      val jDummy = 0 // assert: EscapeHatchDummyLinter
+      var hDummy = 0 // assert: EscapeHatchDummyLinter
+    }
+  }
+
+  // Type
+  @SuppressWarnings(Array("EscapeHatchDummyLinter"))
+  type DummyType_1 = Any
+
+  // Trait
+  @SuppressWarnings(Array("EscapeHatchDummyLinter"))
+  trait DummyTrait_1 {
+    def aDummy = ()
+  }
+
+  // Object
+  @SuppressWarnings(Array("EscapeHatchDummyLinter"))
+  object DummyObject_1 {
+    def aDummy = ()
+  }
+
+  // Class
+  @SuppressWarnings(Array("EscapeHatchDummyLinter"))
+  class DummyClass_1(val aDummy: Int, val bDummy: Int) {
+
+    val cDummy = 0
+    var dDummy = 0
+
+    def this(eDummy: String, fDummy: String) {
+      this(eDummy.toInt, fDummy.toInt)
+    }
+
+    def gDummy(hDummy: Int, iDummy: Int): Unit = {
+      val jDummy = 0
+      var hDummy = 0
+    }
+  }
+
+  // Primary constructor
+  class DummyClass_2 @SuppressWarnings(Array("EscapeHatchDummyLinter"))( // assert: EscapeHatchDummyLinter
+                 val aDummy: Int, val bDummy: Int) {
+
+    val cDummy = 0 // assert: EscapeHatchDummyLinter
```

Comment:
I think it's sufficient to have only a single `val cDummy = 0` in this template to test the scoping of the annotations. The other expressions are redundant.

Reply:
done